web/themes/new_weather_theme/assets/js/components/locationSearch.js

```diff
@@ -1,3 +1,123 @@
+const goToLocationPage = (latitude, longitude) => {
+  // Push the next URL into the history to preserve the current URL in the
+  // browser history stack.
+  window.history.pushState(null, null, `/point/${latitude}/${longitude}`);
+  window.location.href = `/point/${latitude}/${longitude}`;
+};
```

Comment:
I'm not opposed. Especially since the server is going to make the call to the `/points` endpoint again anyway to get the place name. (It's very silly...)